gdp-src-build/conf/templates/dragonboard-410c.local.conf

```diff
@@ -0,0 +1,5 @@
+include templates/local.inc
+
+MACHINE = "dragonboard-410c"
+
+ACCEPT_EULA_dragonboard-410c = "1"
```

Comment:
Just my (unasked for :) ) 2c but I would either respect (mirror) what the underlying YBSP does or ask the originator about your plan. I'm no legal expert but the act of removing the acceptance act from the end user may constitute a defence for violating the license. It will likely also have consequences for any binary distribution of a build.